recipes/numpy-stl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "numpy-stl" %}
+{% set version = "2.7.0" %}
+{% set sha256 = "ede911118cfee5a8fd4c341b418fc55bfcd70a557686febc4efb6693297e3aa2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - stl = stl.main:main
+    - stl2ascii = stl.main:to_ascii
+    - stl2bin = stl.main:to_binary
+
+requirements:
+  build:
+    - setuptools
```

Comment:
I think setuptools should be in the host section.

Reply:
Okay, moved it to the `host` section. Sorry, my first conda-build 3 recipe!